tests/test_physics/test_ansatz.py

```diff
@@ -155,3 +154,102 @@ def test_PolyExpAnsatz_simplify_v2(Abc):
     assert len(ansatz.A) == 1
     assert len(ansatz.b) == 1
     assert np.allclose(ansatz.c, 2 * c)
+
+
+class TestArrayAnsatz:
+    r"""Tests all algebra related to ArrayAnsatz."""
+
+    def test_ArrayAnsatz_init_(self):
```

Comment:
`test_ArrayAnsarz_*` --> `test_*` for here and all the other ones? It's clear enough, since they are inside the class